team1/colors.txt

```diff
@@ -1,14 +1,12 @@
-
-Red
+Beige
+Black
```

Comment:
Black is my sponsor's favorite color!

Can you redo this pull request without deleting Black?

Reply:
Black is still there.
Don you see the "+" in front of the color ?